style.css

```diff
@@ -649,3 +669,51 @@ header i {
   align-self: center;
   margin: 0 42px 0 0;
 }
+
+.contact-form {
+  display: flex;
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/SolomonTheCoder/Microverse-Portfolio/pull/3#discussion_r1150922251) please make sure to follow the CSS best practices 💪 and avoid repeating `display: flex` property 🎯 This can be done by having a class called `.flex-row` or `.flex-column` then assigning it wherever possible as a class name in your html markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
To learn more about it, please follow [this article](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 🔗